code/game/machinery/computer/cloning.dm

```diff
@@ -532,11 +545,7 @@
 		scantemp = "<font class='bad'>Unable to locate valid genetic data.</font>"
 		playsound(src, 'sound/machines/terminal_prompt_deny.ogg', 50, 0)
 		return
-	if(NO_DNA_COPY in dna.species.species_traits)
-		scantemp = "<font class='bad'>The DNA of this lifeform could not be read due to an unknown error!</font>"
-		playsound(src, 'sound/machines/terminal_prompt_deny.ogg', 50, 0)
-		return

```

Comment:
Almost missed this. We have this code to prevent IPC cloning. Add it back, please.

Reply:
ah, ok